src/resources/ble_esp32.cc

```diff
@@ -868,6 +874,17 @@ BleCharacteristicResource::~BleCharacteristicResource() {
   service_->remove_characteristic(this);
 }
 
+uint16 BleCharacteristicResource::get_mtu() {
+  int min_sub_mtu = -1;
```

Comment:
Code that was copied from the `get_att_mtu` primitive.
Tbh, I don't really understand it.